### PR TITLE
chore(connectors): add deprecation notice for some message start event templates

### PR DESCRIPTION
**Pull request by chillleader**
## Description

This PR will add a deprecation notice about none start event element templates to the announcements page.

https://github.com/camunda/team-connectors/issues/857

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [x] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [ ] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/reference/announcements.md [hunk 1]
```diff
@@ -91,7 +91,17 @@ Version changes are made to supported environments:
 
 To learn more about supported environments, see [supported environments](/reference/supported-environments.md).
 
-### Breaking changes in the Connector SDK
+### Connectors
```
Comment (christinaausley) [inline docs/reference/announcements.md:94]: Given the terminology of element template here, is this applicable only to Desktop Modeler? If so, we should likely also add a note in that documentation.

Comment (chillleader) [reply docs/reference/announcements.md:94]: Actually it's applicable to both Desktop and Web Modelers - Web Modeler users will no longer see the "plain start event" connector option for Kafka, RMQ, SQS, and SNS when appending a template to an event.

Comment (christinaausley) [reply docs/reference/announcements.md:94]: Given this is not exclusive to Desktop Modeler then, should element template be called Connector template in these instances?

**Comment by christinaausley** [comment]
CC @sbuettner for review prior to release.

File: docs/reference/announcements.md (hunk 1 above)
Comment (sbuettner) [reply docs/reference/announcements.md:96]: @christinaausley AFAIK element template is the way to go.

**Review by sbuettner — APPROVED** [review]

**Comment by akeller** [comment]
Hey y'all, as this was merged it's only on `/next/` or 8.7. @christinaausley can you please help with a backport so it's part of 8.6 too?

**Pull request merged**